libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -34,6 +34,7 @@
             <button
               mat-flat-button
               color="primary"
+              [attr.aria-label]="'click to add' + b.title + 'authored by'+ b.authors.join(',')+ ' to the reading list'"
```

Comment:
I like the idea here! The code is structured correctly but from a UX perspective, it's probably too long for a label. I would remove the label or make it something like "Add to my reading list"